examples/nodejs/rate-limiter/README.md

```diff
@@ -0,0 +1,133 @@
+<head>
+  <meta name="Momento Node.js Client Library Documentation" content="Node.js client software development kit for Momento Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+
+## Rate-limiter Requirements
```

Comment:
I would add a section above this describing what a rate limiter is and why someone might want to use one.

Reply:
I also don't see anywhere in here where you are talking about what units of time you are able to limit for, or constraints of this class.

Reply:
Will add. Great feedback!